packages/@aws-cdk/aws-config/lib/rule.ts

```diff
@@ -383,3 +391,356 @@ export class CustomRule extends RuleNew {
+  public static readonly RDS_DB_SNAPSHOT = new ResourceType('AWS::RDS::DBSnapshot');
+
+  /** Amazon RDS database subnet group which is associated with an EC2 security group or VPC */
+  public static readonly RDS_DB_SUBNET_GROUP = new ResourceType('AWS::RDS::DBSubnetGroup');
+
+  /** Amazon RDS event subscription */
+  public static readonly RDS_EVENT_SUBSCRIPTION = new ResourceType('AWS::RDS::EventSubscription');
+  /**
+   * Amazon RDS database cluster which:
+   *  contains an RDS database instance
+   *  is associated with an RDS DB subnet group or EC2 security group
+   */
+  public static readonly RDS_DB_CLUSTER = new ResourceType('AWS::RDS::DBCluster');
+  /** Amazon RDS database cluster snapshot which is associated with an RDS database cluster or a VPC */
+  public static readonly RDS_DB_CLUSTER_SNAPSHOT = new ResourceType('AWS::RDS::EventSubscription');
+
+  /** Transfer acceleration for data over long distances between your client and a bucket.  */
+  public static readonly S3_ATTR_ACCELERATE_CONFIGURATION = new ResourceType('AccelerateConfiguration');
+  /** Access control list used to manage access to buckets and objects. */
+  public static readonly S3_ATTR_BUCKET_ACL = new ResourceType('BucketAcl');
+  /** Policy that defines the permissions to the bucket. */
+  public static readonly S3_ATTR_BUCKET_POLICY = new ResourceType('BucketPolicy');
+  /** Allow cross-origin requests to the bucket. */
+  public static readonly S3_ATTR_CROSS_ORIGIN_CONFIGURATION = new ResourceType('CrossOriginConfiguration');
+  /** Rules that define the lifecycle for objects in your bucket. */
+  public static readonly S3_ATTR_LIFECYCLE_CONFIGURATION = new ResourceType('LifecycleConfiguration');
+  /** Logging used to track requests for access to the bucket. */
+  public static readonly S3_ATTR_LOGGING_CONFIGURATION = new ResourceType('LoggingConfiguration');
+  /** Event notifications used to send alerts or trigger workflows for specified bucket events. */
+  public static readonly S3_ATTR_NOTIFICATION_CONFIGURATION = new ResourceType('NotificationConfiguration');
+  /** Automatic, asynchronous copying of objects across buckets in different AWS Regions. */
+  public static readonly S3_ATTR_REPLICATION_CONFIGURATION = new ResourceType('ReplicationConfiguration');
+  /** Requester pays is enabled.  */
+  public static readonly S3_ATTR_BUCKET_REQUEST_PAYMENT_CONFIGURATION = new ResourceType('RequestPaymentConfiguration');
+  /** Tags added to the bucket to categorize. You can also use tagging to track billing. */
+  public static readonly S3_ATTR_TAGGING_CONFIGURATION = new ResourceType('TaggingConfiguration');
+  /** Static website hosting is enabled for the bucket. */
+  public static readonly S3_ATTR_WEBSITE_CONFIGURATION = new ResourceType('WebsiteConfiguration');
+  /** Versioning is enabled for objects in the bucket. */
+  public static readonly S3_ATTR_VERSIONING_CONFIGURATION = new ResourceType('VersioningConfiguration');
```

Comment:
Not sure those are valid resource types...

Reply:
Yeah, the docs [here](https://docs.aws.amazon.com/config/latest/developerguide/resource-config-reference.html#s3-attributes) aren't 100% clear on how this translates into this scope. Should these all be prefixed with `AWS::S3::Bucket::`, or should this all just be one entry for `AWS::S3::Bucket`?

Reply:
testing them out, i suspect Nick is probably right that there's some prefix needed.

Reply:
ended up dropping these, as they're not selectable from the console either.
tried a few variations (`aws::s3::bucket property` and `aws::s3::bucket::property`).